template/apps/thanos.json

```diff
@@ -2,20 +2,20 @@
 	"categories": [
 		"Other",
 		"Tools",
-		"Minecraft"
+		"Games"
 	],
   "env": [
   	{
   		"name": "TZ",
   		"label": "TZ",
-  		"default": "America/New_York",
-  		"description": "Specify a timezone to use for example Europe/London"
+  		"default": "Europe/London",
+  		"description": "Specify a timezone to use. For example, Europe/London"
   	}
   ],
-	"description": "[Thanos](https://github.com/aternosorg/thanos) is a tool that allows you to shrink your minecraft worlds by deleting unused chuncks.",
+	"description": "Thanos is a tool that allows you to shrink your minecraft worlds by deleting unused chuncks.",
 	"image_amd64": "cristianeduardmihai/thanos:latest",
-	"image_arm64": "cristianeduardmihai/thanos:arm64",
-	"logo": "https://raw.githubusercontent.com/novaspirit/pi-hosted/images/aternos-thanos.png",
+	"image_arm64": "cristianeduardmihai/thanos:latest",
```

Comment:
I don't think this image is cross platform, I know the arm64 tag doesn't exist, but maybe this should be removed all-together, as there's no point including `:latest` for ARM devices if it's not going to work?